docs/src/pages/components/data-grid/sorting/BasicExampleDataGrid.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { DataGrid } from '@mui/x-data-grid';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+const VISIBLE_FIELDS = ['name', 'rating', 'country', 'dateCreated', 'isAdmin'];
+
+export default function BasicExampleDataGrid() {
```

Comment:
I've noticed that country sorting doesn't work in this demo

https://user-images.githubusercontent.com/13808724/150376741-1357d17b-b21e-4cca-bdd3-dfc88538c900.mov

I can reproduce it in storybook as well.

It seemed like an easy fix at first - just a matter of adding `valueGetter`, so that column gets sorted by country label.
It didn't work though.
Adding `valueGetter` broke `renderCell`, since `renderCell` now receives a string (returned by `valueGetter`) instead of whole field object.
I did not expect that and at this point I'm not sure if that's intended.

I can extract it into a separate issue and investigate, so that this PR is not blocked by it.